DataFormats/Parameters/src/AggregatedRunInfo.cxx

```diff
@@ -57,6 +58,80 @@ o2::parameters::AggregatedRunInfo AggregatedRunInfo::buildAggregatedRunInfo(int
   // adjust to the nearest TF edge to satisfy condition (orbitSOR % nOrbitsPerTF == 0)
   orbitSOR = (orbitSOR / nOrbitsPerTF + 1) * nOrbitsPerTF; // +1 to choose the safe boundary ... towards run middle
   orbitEOR = orbitEOR / nOrbitsPerTF * nOrbitsPerTF;
+  // temporary map of orbit shifts for runs <=LHC22m (while waiting for complete run list from CTP/Calib/FirstRunOrbit)
```

Comment:
@ekryshen where these numbers come from? 
The `sorMS` is the STF from the RCT/Info/RunInformation, retrofitted by converting the 
`orbit_reset + run_1st_orbit_from_ctf * LHCOrbitMUS` from \mus to ms. So, during the reverse conversion from STF to 
1st orbit the max rounding error of 999\mus can lead to at most 11 orbits errors.
E.g. for the run 517619 the STF is `1654245241862`, the CTP orbit reset time is `1654239395833483`,  leading to recalculated 1st orbit `65741411`.
The true 1st orbit is 65741421 from
```
alien_find  /alice/data/2022/LHC22c/517619/raw/ o2_ctf_*_tf0000000001.root
/alice/data/2022/LHC22c/517619/raw/1040/o2_ctf_run00517619_orbit0065741421_tf0000000001.root
```
so the error in this case is 10 orbits. Where 109 comes from ?

Reply:
PS actual error is 19 due to the request of `orbitSOR%nOrbitsPerTF == 0`. Actually, for the run 517619 it does not work: the true 1st orbit 65741421 is not multiple of 128. And accidentally, `65741421%128 = 109`, so I am lost...
@lietava is it possible to determine which was the 1st run with request of `orbitSOR%nOrbitsPerTF == 0` ?

Reply:
@shahor02 The offsets were determined empirically using normalized number of contributors per orbit in TF plotted wrt TF start defined according to orbitSOR % nOrbitsPerTF = 0 rule. Here I show an example:
![tf](https://github.com/user-attachments/assets/122f8e48-6c64-4c27-9dee-928c43e47b3c)
(blue region corresponds to the end of TF)
Unfortunatelly I couldn't find the final plot with all runs but I found that practically all runs <=LHC22m are affected. But starting from LHC22o everything seems to be consistent with the rule orbitSOR % nOrbitsPerTF = 0.

Note that I didn't care about true first oribt. The goal was to determine TF borders for TF border cut in the event selection. In that sense current orbitSOR definition is valid with a precision of several TFs but the border of TFs should be precise.

Reply:
@ekryshen but the `orbitSOR % nOrbitsPerTF = 0` does not hold for these runs, it was introduced later.

Reply:
@shahor02, yes, therefore I introduced this empirical shift for selected runs:
orbitSOR += mapOrbitShift[runnumber];

Do I miss something?

Reply:
OK, so you get `65741440 + 109 = 65741549` instead of the true 1st orbit, i.e. effectively shift the start of run by 1 TF.

Fine for me as a temporary hack but I would prefer to retrofit with real value asap (pinging to @lietava )